pkg/kv/kvserver/protectedts/ptpb/protectedts.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package ptpb
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/sql/catalog/descpb"
+)
+
+// MakeRecordClusterTarget returns a target, which when used in a Record, will
+// protect the entire keyspace of the cluster.
+func MakeRecordClusterTarget() *Target {
```

Comment:
[nit] Drop the "Record" in all of these? Now that the protos have been moved out of `message Record`

Reply:
good point, done.